Cargo.lock

```diff
@@ -149,7 +149,7 @@ checksum = "687c230d85c0a52504709705fc8a53e4a692b83a2184f03dae73e38e1e93a783"
 dependencies = [
  "proc-macro2 1.0.21",
  "quote 1.0.7",
- "syn 1.0.41",
+ "syn 1.0.40",
```

Comment:
looks like this lockfile is downgrading deps.

Reply:
Fixed